README.md

```diff
@@ -73,7 +73,14 @@ npm run start
 - `npm run dev:start:with-pagefind` - Starts the local dev environment using exampleSite with working pagefind search
 - `npm run dev:build` - Builds the site using exampleSite
 
+### To run in docker
+
+As memtioned above, fork and clone this repository, run `git submodule`, then run following:
```

Comment:
I believe this should be `mentioned`.

And don't you need more than just `git submodule`. Don't you need to run `git submodule update --init --recursive`?